release-tools/OWNERS

```diff
@@ -2,7 +2,8 @@
 
 approvers:
 - kubernetes-csi-approvers
-- pohly
+- meinhardzhou
```

Comment:
You cannot modify release-tools in the [csi-driver-nvmf](https://github.com/kubernetes-csi/csi-driver-nvmf) repo. You should use it verbatim and only modify files outside of it.

Reply:
Done!